### PR TITLE
Update add-domain

### PR DESCRIPTION
**Pull request by brinhosa**
added reltio-apps.com

## Domain/URL/IP(s) where you have found the Phishing:
[reltio-apps.com/](https://reltio-apps.com/)

## Impersonated domain

reltio.com

## Describe the issue

reltio-apps.com is a SCAM trying to impersonate reltio.com

### Screenshot

<details><summary>Click to expand</summary>

</details>

### CODE REVIEW

File: add-domain [hunk 1]
```diff
@@ -1,3 +1,4 @@
+reltio-apps.com
```
Comment (spirillen) [inline add-domain:1]: IF I may give a suggestion, then add this record to `add-wildcard-domain` for better protection, as it appears the domain is only to be used for evil.

**Comment by spirillen** [comment]
The Chinese translates into

`您的域名指向了CDN节点，但配置未生效或者未在CDN配置此域名。` = `Your domain name points to a CDN node, but the configuration does not take effect or the domain name is not configured on CDN.`

`请到CDN后台添加此域名，或联系管理员处理。` `Please go to the CDN backend to add this domain name, or contact the administrator for processing.`

Which makes the real domain to blacklist `kbpx4fdv.svip.tesucdn.com.` but this is unly usefull for RPZ aware blacklists

```bash
reltio-apps.com.        172800  IN      NS      ns1.dns.com.
reltio-apps.com.        172800  IN      NS      ns2.dns.com.
reltio-apps.com.        600     IN      CNAME   kbpx4fdv.svip.tesucdn.com.
```

```
<body>
  <div id="cf-wrapper">
    <div id="cf-error-details" class="p-0">
      <header class="mx-auto pt-10 lg:pt-6 lg:px-8 w-240 lg:w-full mb-15 antialiased">
         <h1 class="inline-block md:block mr-2 md:mb-2 font-light text-60 md:text-3xl text-black-dark leading-tight">
           <span data-translate="error">Error</span>
           <span>530</span>
         </h1>
         <span class="inline-block md:block heading-ray-id font-mono text-15 lg:text-sm lg:leading-relaxed">您的IP: 185.129.61.4 &bull;</span>
         <span class="inline-block md:block heading-ray-id font-mono text-15 lg:text-sm lg:leading-relaxed">节点IP: 10.223.0.14</span>
        <h2 class="text-gray-600 leading-1.3 text-3xl lg:text-2xl font-light">域名未配置</h2>
      </header>

      <section class="w-240 lg:w-full mx-auto mb-8 lg:px-8">
          <div id="what-happened-section" class="w-1/2 md:w-full">
            <h2 class="text-3xl leading-tight font-normal mb-4 text-black-dark antialiased" data-translate="what_happened">什么问题?</h2>
            <p>您的域名指向了CDN节点，但配置未生效或者未在CDN配置此域名。</p>
            
          </div>

          
          <div id="resolution-copy-section" class="w-1/2 mt-6 text-15 leading-normal">
            <h2 class="text-3xl leading-tight font-normal mb-4 text-black-dark antialiased" data-translate="what_can_i_do">如何解决?</h2>
            <p>请到CDN后台添加此域名，或联系管理员处理。</p>
          </div>
          
      </section>

      <div class="cf-error-footer cf-wrapper w-240 lg:w-full py-10 sm:py-4 sm:px-8 mx-auto text-center sm:text-left border-solid border-0 border-t border-gray-300">

</div><!-- /.error-footer -->


    </div><!-- /#cf-error-details -->
  </div><!-- /#cf-wrapper -->


</body>
```

File: add-domain (hunk 1 above)
Comment (brinhosa) [reply add-domain:1]: Thanks for the suggestion, just added.

Comment (spirillen) [reply add-domain:1]: So this issue can be closed?

**Comment by spirillen** [comment]
Closing as no respons

**Pull request closed without merging**